Telerik.JustMock/Core/MocksRepository.cs

```diff
@@ -1552,15 +1554,25 @@ select new
 
             var resultList = resultsQ.ToList();
 
-            var nonSequential = resultList.Where(x => !x.MethodMock.IsSequential && x.Acceptable).LastOrDefault();
-            if (nonSequential != null)
-                return nonSequential.MethodMock;
+            var nonSequentialOrInOrder = resultList.Where(x => !x.MethodMock.IsSequential && !x.MethodMock.IsInOrder && x.Acceptable).LastOrDefault();
+            if (nonSequentialOrInOrder != null)
+            {
+                return nonSequentialOrInOrder.MethodMock;
+            }
+
+            var inOrder = resultList.Where(x => x.MethodMock.IsInOrder && !x.MethodMock.IsUsed && x.Acceptable).FirstOrDefault();
```

Comment:
Is it possible that the method mock is in order, it is not used and it is sequential?